x-pack/plugins/security_solution/public/common/components/exceptions/viewer/translations.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const EXCEPTION_NO_SEARCH_RESULTS_PROMPT_BODY = i18n.translate(
+  'xpack.securitySolution.exceptions.viewer.noSearchResultsPromptBody',
+  {
+    defaultMessage: 'No search results found',
+  }
+);
+
+export const EXCEPTION_EMPTY_PROMPT_TITLE = i18n.translate(
+  'xpack.securitySolution.exceptions.viewer.addExceptionsEmptyPromptTitle',
+  {
+    defaultMessage: 'Add exceptions to this rule',
+  }
+);
+
+export const EXCEPTION_EMPTY_PROMPT_BODY = i18n.translate(
+  'xpack.securitySolution.exceptions.viewer.emptyPromptBody',
+  {
+    defaultMessage: 'There are no exceptions on your rule. Create your first rule exception.',
```

Comment:
Is it possible to enter line breaks or hard returns in these strings? It'd be nice to break between the sentences so the lines are about the same length, but understandable if that's not possible. 
```suggestion
    defaultMessage: 'There are no exceptions for this rule. Create your first rule exception.',
```

Reply:
Updated text! Not too sure about adding line breaks as that might interfere with the responsiveness of the component.

Reply:
@yctercero I figured that'd be the case, just thought I'd ask. Thanks, LGTM!